vowpalwabbit/example.cc

```diff
@@ -185,7 +182,7 @@ flat_example* flatten_example(vw& all, example* ec)
 {
   flat_example& fec = calloc_or_throw<flat_example>();
   fec.l = ec->l;
-  fec.weight = ec->weight;
```

Comment:
should this be removed? From what I understand, example.weight is still used?

Reply:
Yes, this was added in the pred/learn PR and so is undone by these changes